homeassistant/components/kaiterra/sensor.py

```diff
@@ -58,6 +60,8 @@ def __init__(self, api, name, device_id, description: SensorEntityDescription):
         self._device_id = device_id
         self.entity_description = description
         self._attr_name = f"{name} {description.name}"
+        if TYPE_CHECKING:
+            assert description.name is not None
```

Comment:
The `name` property is always defined in the descriptions above, but maybe we need to adjust this to be future-proof?

Reply:
This would require a mixin to change the type of the `name` attribute in the description.

Reply:
It's a bigger change than originally anticipated - but I think the mixin is good. I couldn't override `name` though so I created `suffix` instead.